pkg/services/ngalert/api/api_ruler.go

```diff
@@ -421,21 +415,6 @@ func (srv RulerSrv) updateAlertRulesInGroup(c *contextmodel.ReqContext, groupKey
 		return ErrResp(http.StatusInternalServerError, err, "failed to update rule group")
 	}
 
-	for _, rule := range finalChanges.Update {
```

Comment:
I don't see any test that assert the fact that we no longer issues updates with `UpdateAlertRule` and `DeleteAlertRule`. Can we add that or flip the conditional around the existing ones?

Reply:
The methods in the scheduler will be deleted as well as any reference to the scheduler in API layer n a follow-up PR and therefore there is no need for those tests

Reply:
I will delete the code in thins PR. I think it will not complicate backport.

Reply:
Agreed.